samples/sampler/stories/qa/Panel.js

```diff
@@ -0,0 +1,84 @@
+import Button from '@enact/sandstone/Button';
+import Item from '@enact/sandstone/Item';
+import {Header, Panels, Panel} from '@enact/sandstone/Panels';
+import Spotlight from '@enact/spotlight';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {useState, useCallback} from 'react';
+import compose from 'ramda/src/compose';
+
+export const WithAutoFocusControl = () => {
+	// hooks
+	const initialState = 0;
+	const [panelIndex, setState] = useState(initialState);
+
+	const forward = useCallback(() => {
+		setState(panelIndex + 1);
+	}, [panelIndex]);
+
+	const backward = useCallback(() => {
+		setState(panelIndex - 1);
+	}, [panelIndex]);
+
+	const handleTransition = useCallback(() => {
+		setTimeout(() => {
+			Spotlight.focus(`panel-container-${panelIndex}`);
+		}, 1000);
```

Comment:
```suggestion
		setTimeout(() => {
			if(!Spotlight.getPointerMode() && !Spotlight.getCurrent() && !Spotlight.isPaused()) {
				Spotlight.focus(`panel-container-${panelIndex}`);
			}
		}, 1000);
```
Some condition is needed when we call `Spotlight.focus()`.

Reply:
It's added.